lldb/test/API/functionalities/data-formatter/type-recognizer/value-object-cast/RecognizerFormatter.py

```diff
@@ -0,0 +1,74 @@
+from lldb import SBType, SBValue
+
+def generalHandler(value: SBValue, internal_dict) -> SBType:
+    supported_targets = [
+        "A_1",
+        "A_2",
+
+        "B_2",
+        "B_3",
+    ]
+
+    # Get target type from value data members
+    assert value.GetChildMemberWithName("target").IsValid()
```

Comment:
Have you tested that failed assertions and raised exceptions would indeed fail the test?

Reply:
Yes, it fails.